Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import AdyenNetworking
+import AdyenSession
+import UIKit
+
+internal final class DropIn: DropInExampleProtocol {
+
+    // MARK: - Properties
+
+    internal var dropInComponent: PresentableComponent?
+    internal weak var presenter: PresenterExampleProtocol?
+    internal var paymentMethods: PaymentMethods?
+    internal var session: AdyenSession?
+    private lazy var containerView = UIView()
+    private lazy var loadingView = LoadingView(contentView: containerView)
+
+    // MARK: - Initializers
+
+    internal init() {}
+
+    // MARK: - Networking
+
+    internal func requestInitialData() {
+        requestAdyenSessionConfiguration { adyenSessionConfig, errorResponse in
+            guard let config = adyenSessionConfig else {
+                return
+            }
+            AdyenSession.initialize(with: config,
+                                    delegate: self,
+                                    presentationDelegate: self) { [weak self] result in
+                switch result {
+                case let .success(session):
+                    self?.session = session
+                    self?.paymentMethods = session.sessionContext.paymentMethods
+                    self?.loadingView.showsActivityIndicator = false
+                case let .failure(errorResponse):
+                    self?.presentAlert(with: errorResponse)
+                    self?.loadingView.showsActivityIndicator = false
+                }
+            }
+        }
+    }
+
+    internal func presentDropInComponentSession() {
+        guard let dropIn = dropInComponent(from: paymentMethods) else { return }
+
+        dropIn.delegate = session
+        dropIn.partialPaymentDelegate = session
+        dropInComponent = dropIn
+
+        presenter?.present(viewController: dropIn.viewController, completion: nil)
+    }
+
+    internal func dropInComponent(from paymentMethods: PaymentMethods?) -> DropInComponent? {
+        guard let paymentMethods = paymentMethods else { return nil }
+
+        let configuration = DropInComponent.Configuration()
+
+        if let applePayPayment = try? ApplePayPayment(payment: ConfigurationConstants.current.payment,
+                                                      brand: ConfigurationConstants.appName) {
+            configuration.applePay = .init(payment: applePayPayment,
+                                           merchantIdentifier: ConfigurationConstants.applePayMerchantIdentifier)
+        }
+
+        configuration.actionComponent.threeDS.delegateAuthentication = ConfigurationConstants.delegatedAuthenticationConfigurations
+        configuration.actionComponent.threeDS.requestorAppURL = URL(string: ConfigurationConstants.returnUrl)
```

Comment:
We have already set this to Sessions call 🤔
https://github.com/Adyen/adyen-ios/pull/1184/files#diff-05c97209b181bb6a898a57990a209ee18b8ef1e4784b45c72b57652247f82194R50-R54

Reply:
yes, I have removed this. 0f782b0d3cbf890eea056cb7060b41c496e6955e